src/main/jjtree/net/sf/jsqlparser/parser/JSqlParserCC.jjt

```diff
@@ -1751,14 +1725,15 @@ String RelObjectNameWithoutStart() :
     (result = RelObjectNameWithoutValue() | tk=<K_TOP> | tk=<K_VALUE> | tk=<K_VALUES>
         | tk=<K_INTERVAL> )
 
-    {
-		if (tk!=null) result=tk.image;
-		return result;
-	}
+    { return tk!=null ? tk.image : result; }
 }
 
 /*
 Extended version of object names.
+
+These tokens can be used as names for Schema and Tables and Columns
+BUT NOT for Aliases (without quoting)
```

Comment:
:+1:  useful comment!

Reply:
Maybe a more descriptive name of the `RelObjNameExt` would be:
`SchemaTableColumnName`